pkg/apis/core/v1alpha1/filewatch_types.go

```diff
@@ -99,6 +105,15 @@ func (in *FileWatchList) GetListMeta() *metav1.ListMeta {
 
 // FileWatchStatus defines the observed state of FileWatch
 type FileWatchStatus struct {
+	LastEventTime *metav1.Time `json:"lastEventTime,omitempty"`
+	SeenFiles     []string     `json:"seenFiles,omitempty"`
+	ErrorMessage  string       `json:"errorMessage"`
+	// FileEvents []FileEvent `json:"fileEvents,omitempty"`
+}
+
+type FileEvent struct {
```

Comment:
is this used?

Reply:
No - I started down this path and then removed it to favor data model simplicity, but I think it _might_ end up being necessary so that consumers watching can get "files changed since" semantics. Deleted it for now, we can reintroduce if the need arises once the controller is being used